ansible_collections/arista/avd/roles/eos_designs/templates/network_services/logic.j2

```diff
@@ -85,6 +85,29 @@
                                                                            svi_profile.structured_config,
                                                                            svi_parent_profile.structured_config)}) %}
 {%                                 do tmp_svi.setdefault("bgp", {}).update({"structured_config": bgp_svi_struct_cfg}) %}
+{%                                 set tmp_svi_node = tmp_svi.pop("nodes", {}) | arista.avd.convert_dicts('name') |
+                                                                                 selectattr('name', 'arista.avd.defined', inventory_hostname) |
+                                                                                 first %}
+{%                                 if tmp_svi_node is arista.avd.defined %}
```

Comment:
We could support setting any setting per-node if we just combine the node config after line 80. I would suggest implementing that in stead of this very specific logic.